### DISCUSSION TITLE
Block level filtering within current page

### DISCUSSION THREAD

**Discussion by tonyxiao** [vslinko/obsidian-outliner#400] [Ideas]
Roam has the ability for you to filter the current list within a page by a specific backlink or tag. I am really missing it and would love to see it in Obsidian. Is that possible to do via the outliner? I'd be happy to contribute code to make this possible if someone can give me some pointers. 

PS: Posted here https://github.com/vslinko/obsidian-outliner/issues/399 but realized it was classified as a bug. So here again

**Comment by vslinko** [comment]
ref: https://github.com/vslinko/obsidian-zoom/discussions/85
